File: 724-find-pivot-index/724-find-pivot-index.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
        int total = 0;
        int n = 0;
        for (int i = 0; i < nums.size(); i++) total += nums[i];
        for (int i = 0; i < nums.size(); i++){
            if(n == total - n - nums[i]) return i;
            n += nums[i];
        }
        return -1;
    }
};

File: 724-find-pivot-index/724-find-pivot-index.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        total = sum(nums)
        n = 0
        for i in range(len(nums)):
            if (n == (total - nums[i] - n)):
                return i
            n += nums[i]
        return -1    
        
        